api/migrations/schema/Version20230409164830.php

```diff
@@ -0,0 +1,108 @@
+                    date_trunc(\'second\', now()::timestamp), date_trunc(\'second\', now()::timestamp)
+            FROM    camp c
+            WHERE   c.id = \'e5027d852487\'
+        ');
+
+        // Prod:
+        // J+S Lager (Deutsch)
+        $this->addSql('INSERT INTO activity_progress_label (id, campId, position, title, createTime, updateTime) 
+            SELECT  encode(gen_random_bytes(6), \'hex\'), \'75b3572a338e\', 1, \'In Planung\', 
+                    date_trunc(\'second\', now()::timestamp), date_trunc(\'second\', now()::timestamp)
+            FROM    camp c
+            WHERE   c.id = \'75b3572a338e\'
+            UNION
+            SELECT  encode(gen_random_bytes(6), \'hex\'), \'75b3572a338e\', 2, \'Geplant\', 
+                    date_trunc(\'second\', now()::timestamp), date_trunc(\'second\', now()::timestamp)
+            FROM    camp c
+            WHERE   c.id = \'75b3572a338e\'
+            UNION
+            SELECT  encode(gen_random_bytes(6), \'hex\'), \'75b3572a338e\', 3, \'Lagerleiter:in OK\', 
+                    date_trunc(\'second\', now()::timestamp), date_trunc(\'second\', now()::timestamp)
+            FROM    camp c
+            WHERE   c.id = \'75b3572a338e\'
+            UNION
+            SELECT  encode(gen_random_bytes(6), \'hex\'), \'75b3572a338e\', 4, \'Coach OK\', 
+                    date_trunc(\'second\', now()::timestamp), date_trunc(\'second\', now()::timestamp)
+            FROM    camp c
+            WHERE   c.id = \'75b3572a338e\'
+        ');
+
+        // Camp J+S (français)
+        $this->addSql('INSERT INTO activity_progress_label (id, campId, position, title, createTime, updateTime) 
+            SELECT  encode(gen_random_bytes(6), \'hex\'), \'497f974e7d5d\', 1, \'En cours de planification\', 
+                    date_trunc(\'second\', now()::timestamp), date_trunc(\'second\', now()::timestamp)
+            FROM    camp c
+            WHERE   c.id = \'497f974e7d5d\'
+            UNION
+            SELECT  encode(gen_random_bytes(6), \'hex\'), \'497f974e7d5d\', 2, \'Planifié\', 
+                    date_trunc(\'second\', now()::timestamp), date_trunc(\'second\', now()::timestamp)
+            FROM    camp c
+            WHERE   c.id = \'497f974e7d5d\'
+            UNION
+            SELECT  encode(gen_random_bytes(6), \'hex\'), \'497f974e7d5d\', 3, \'Chef de camp OK\', 
+                    date_trunc(\'second\', now()::timestamp), date_trunc(\'second\', now()::timestamp)
+            FROM    camp c
+            WHERE   c.id = \'497f974e7d5d\'
+            UNION
+            SELECT  encode(gen_random_bytes(6), \'hex\'), \'497f974e7d5d\', 4, \'Coach OK\', 
+                    date_trunc(\'second\', now()::timestamp), date_trunc(\'second\', now()::timestamp)
+            FROM    camp c
+            WHERE   c.id = \'497f974e7d5d\'
+        ');
+
+        // All ohter camps
+        $this->addSql('INSERT INTO activity_progress_label (id, campId, position, title, createTime, updateTime) 
```

Comment:
We should test this on staging with some a recent DB dump, right?

Reply:
That would be great :)